### PR TITLE
Changed the flintlock recipe

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
Adapted the .44 cap and ball file to become the flintlock file; removed the plastic bag from the recipe; added an uncraft recipe as requested by KillyDigiMortal on the forums: http://smf.cataclysmdda.com/index.php?topic=5865.msg288388#msg288388 ; and changed steel entry to lead in the ammo entry.

Also lowered the smokeless powder requirement by half.

### CODE REVIEW

File: data/json/requirements/ammo.json [hunk 1]
```diff
@@ -15,5 +15,12 @@
       [ [ "9mm_casing", 1 ] ],
       [ [ "smpistol_primer", 1 ] ]
     ]
+  },
+  {
+    "id": "ammo_flintlock",
```
Comment (mugling) [inline data/json/requirements/ammo.json]: This is obfuscation. You would be better just using `paper `in the `recipe`

**Comment by mugling** [comment]
> Adapted the .44 cap and ball file to become the flintlock file

I don'tr see this part?

**Comment by BorkBorkGoesTheCode** [comment]
I just copied the recipes/ammo/pstol file, and added the flintlock ammo to it. The recipe uses paper instead of ammo_flintlock now.

**Comment by mugling** [comment]
JSON load errors:
```
src/requirements.cpp:33 [const requirement_data &string_id<requirement_data>::obj() const
[T = requirement_data]] Tried to get invalid requirements: ammo_flintlock
```

**Comment by DangerNoodle** [comment]
Why did you lower the smokeless powder requirement, but not black powder? Reasoning for change in https://github.com/CleverRaven/Cataclysm-DDA/pull/18896 was given:

> This rebalances the amount of black gunpowder used in ammunition recipes. The original standard for blackpowder did not account for the fact that a unit of blackpowder weighs five times as much as a unit of normal powder. Five times the amount should realistically be well in excess of the amount needed to make up for the lower energy density of blackpowder. Additionally, in gameplay terms it is often difficult to produce blackpowder in quantity.

Change in this pull request created inconsistency with all other recipes that allow use of black powder.

**Pull request merged**